src/screens/settings/SettingsBackupScreen/index.tsx

```diff
@@ -38,57 +39,85 @@ const BackupSettings = ({ navigation }: BackupSettingsScreenProps) => {
     <>
       <ScreenContainer theme={theme}>
         <Appbar
-          title="Backup"
+          title={getString('moreScreen.settingsScreen.backup')}
           handleGoBack={() => navigation.goBack()}
           theme={theme}
         />
         <ScrollView style={{ paddingBottom: 40 }}>
           <List.Section>
-            <List.SubHeader theme={theme}>Remote Backup</List.SubHeader>
+            <List.SubHeader theme={theme}>
+              {getString('moreScreen.settingsScreen.backupScreen.remoteBackup')}
+            </List.SubHeader>
             <List.Item
-              title="Self Host"
-              description="Backup to your server"
+              title={getString(
+                'moreScreen.settingsScreen.backupScreen.selfHost',
+              )}
+              description={getString(
+                'moreScreen.settingsScreen.backupScreen.selfHostDesc',
+              )}
               theme={theme}
               onPress={openSelfHostModal}
               disabled={Boolean(hasAction)}
             />
 
             <List.Item
-              title="Googe Drive"
-              description="Backup to your Google Drive"
+              title={getString(
+                'moreScreen.settingsScreen.backupScreen.googeDrive',
+              )}
+              description={getString(
+                'moreScreen.settingsScreen.backupScreen.googeDriveDesc',
+              )}
               theme={theme}
               onPress={openGoogleDriveModal}
               disabled={Boolean(hasAction)}
             />
-            <List.SubHeader theme={theme}>Legacy Backup</List.SubHeader>
+            <List.SubHeader theme={theme}>
+              {getString('moreScreen.settingsScreen.backupScreen.legacyBackup')}
+            </List.SubHeader>
             <List.Item
-              title="Create backup (Deprecated)"
-              description="Can be used to restore current library"
+              title={getString(
+                'moreScreen.settingsScreen.backupScreen.createBackupDeprecated',
+              )}
+              description={getString(
+                'moreScreen.settingsScreen.backupScreen.createBackupDeprecatedDesc',
+              )}
```

Comment:
We can add deprecated to common

Reply:
But in this case it won't make this simpler.

Reply:
We can keep `deprecated` translation for future usage. Like we can create `deprecated` label